landoapi/repos.py

```diff
@@ -72,6 +82,8 @@ class Repo:
     approval_required: bool = False
     commit_flags: list[tuple[str, str]] = field(default_factory=list)
     config_override: dict = field(default_factory=dict)
+    codefreeze_enabled: bool = False
+    product_details_url: str = ""
```

Comment:
Will `codefreeze_enabled` ever be `True` without `product_details_url` being set to a non-empty string? Maybe we could get rid of the boolean and just do `if repo.product_details_url` instead.

Reply:
in this situation no, but if we were to use this url for other repos in the future, then it wouldn't be a reliable way to tell. however, we could just check for the `NEXT_CODEFREEZE_DATE` key in the returned json for the given product